### PR TITLE
Optimize some repo metastuff and prepare v1.9.0

### LINKED ISSUE #357: Documentation on readthedocs is still for version 1.3.0
It'd be nice if we could have the docs for v1.8.0 up there. Even if there aren't significant changes, it'd probably make people feel better about using those docs

### PR DESCRIPTION
**Pull request by nir0s**
Nothing too fancy. See commit messages.

btw, just noting that codecov doesn't work and that there are some discussions about this: https://github.com/codecov/codecov-action/issues/844

Also fixes #357

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+## 1.9.0 (2023.12.19)
+
+ENHANCEMENTS:
+* Refactor distro.info() method to return an InfoDict [[#360](https://github.com/python-distro/distro/pull/360)]
+* Ignore the file '/etc/board-release' [[#353](https://github.com/python-distro/distro/pull/353)]
+* Ignore the file '/etc/ec2_version' [[#359](https://github.com/python-distro/distro/pull/353)]
+
+RELEASE:
+* Run Python 3.6 on Ubuntu 20.04 for CI and bump isort [[#355](https://github.com/python-distro/distro/pull/355)]
+
```
Comment (HorlogeSkynet) [inline CHANGELOG.md:10]:
```suggestion
```

Unnecessary new line.

File: Makefile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015,2016 Nir Cohen
+# Copyright 2015-2023 Nir Cohen
```
Comment (HorlogeSkynet) [inline Makefile:1]:
```suggestion
# Copyright 2015-2021 Nir Cohen
# Copyright 2021-2023 python-distro/maintainers
```

(just an idea)

File: src/distro/distro.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-# Copyright 2015,2016,2017 Nir Cohen
+# Copyright 2015-2023 Nir Cohen
```
Comment (HorlogeSkynet) [inline src/distro/distro.py:2]:
```suggestion
# Copyright 2015-2021 Nir Cohen
# Copyright 2021-2023 python-distro/maintainers
```

(same)

File: tox.ini [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015,2016 Nir Cohen
+# Copyright 2015-2023 Nir Cohen
```
Comment (HorlogeSkynet) [inline tox.ini:1]:
```suggestion
# Copyright 2015-2021 Nir Cohen
# Copyright 2021-2023 python-distro/maintainers
```

(same again)

File: src/distro/distro.py [hunk 5]
```diff
@@ -55,7 +55,7 @@
     # Python 3.7
     TypedDict = dict
 
-__version__ = "1.8.0"
+__version__ = "1.9.0"
```
Comment (HorlogeSkynet) [inline src/distro/distro.py:58]: I've got a notification about a `v1.9.0` tag being created, but no release was pushed on PyPI : have you planned to delete/recreate this tag when this PR will be merged to `master` for consistency ?

**Review by HorlogeSkynet — COMMENTED** [review]
Thanks, bye :wave:

File: Makefile (hunk 2 above)
Comment (nir0s) [reply Makefile:1]: Good point.

File: src/distro/distro.py (hunk 5 above)
Comment (nir0s) [reply src/distro/distro.py:58]: Yeah, that was my bad. I created the tag and pushed it to test some deployment mechanism I was testing, but pushed it and forgot to delete it. I'll delete the tag now, fix, merge, then push the tag.

**Comment by nir0s** [comment]
@HorlogeSkynet, I added a `.readthedocs.yaml` config file so that we can start building docs again.

**Comment by HorlogeSkynet** [comment]
Awewome, this will fix #357 I assume.

**Comment by nir0s** [comment]
> Awewome, this will fix #357 I assume.

Indeed. I mentioned that in the PR comment.

**Comment by nir0s** [comment]
Also wik, we need additional maintainers in RTD.

**Review by HorlogeSkynet — COMMENTED** [review]
LGTM, _modulo_ my previous picky suggestions :+1:

**Comment by nir0s** [comment]
> LGTM, _modulo_ my previous picky suggestions 👍

WDYM? I fixed all.

**Comment by HorlogeSkynet** [comment]
> > LGTM, _modulo_ my previous picky suggestions 👍
> 
> WDYM? I fixed all.

Sorry my bad, there is just one little dangling suggestion for an empty newline in CHANGELOG :pray:

File: CHANGELOG.md (hunk 1 above)
Comment (HorlogeSkynet) [reply CHANGELOG.md:10]: Okay and sorry then, I haven't checked out your branch :pray:

**Review by HorlogeSkynet — APPROVED** [review]

**Pull request merged**